db/migrate/20160520175340_add_host_applicable_package.rb

```diff
@@ -0,0 +1,8 @@
+class AddHostApplicablePackage < ActiveRecord::Migration
+  def change
+    create_table "katello_content_facet_applicable_rpms" do |t|
+      t.references 'content_facet', :null => false
+      t.references 'rpm', :null => false
+    end
```

Comment:
Should the combination of content_facet -> rpm be unique? If so, what about adding a unique index to this migration?